### ISSUE TITLE
FCOS3D Model on nuScenes Test set

### ISSUE THREAD

**Issue by Avi-JS** [open-mmlab/mmdetection3d#997]
Hi,

Out of the pretrained FCOS3D models provided in [configs/fcos3d](https://github.com/open-mmlab/mmdetection3d/tree/master/configs/fcos3d), which one generated the scores reported on the nuScenes Leaderboard (with a reported mAP of 0.358)? 

Also, from the [logs](https://download.openmmlab.com/mmdetection3d/v0.1.0_models/fcos3d/fcos3d_r101_caffe_fpn_gn-head_dcn_2x8_1x_nus-mono3d_finetune/fcos3d_r101_caffe_fpn_gn-head_dcn_2x8_1x_nus-mono3d_finetune_20210717_095645.log.json) of the models in configs/fcos3d, I see that the best model achieves a validation mAP of 32.1, while the FCOS3D model on the leader board achieves 35.8 on the test set. If this was the model that was submitted to the leaderboard, does it mean it is easier to generalise to the test set than the validation set?

Thanks!

**Comment by Tai-Wang** [comment]
Hi @Avi-JS , we only provide models trained with 1x (12 epochs) preliminary training and 1x fine-tuning. The final models on the leaderboard are trained with double GPUs and double epochs. So the performance seems a little better. You can refer to the ablation study in our [paper](https://arxiv.org/pdf/2104.10956.pdf).

As for the results on the validation and test set, the final model on the test set benefits a little from a simple model ensemble and fine-tuning on the validation set as presented in the entry of the [leaderboard](https://www.nuscenes.org/object-detection?externalData=all&mapData=all&modalities=Camera). The basic performance should be similar (maybe a little better on the test in my memory).
